gnovm/pkg/gnomod/pkg.go

```diff
@@ -165,3 +167,94 @@ func (sp SortedPkgList) GetNonDraftPkgs() SortedPkgList {
 	}
 	return res
 }
+
+// SubPkgsFromPaths returns a list of subpackages from the given paths.
+func SubPkgsFromPaths(paths []string) ([]*SubPkg, error) {
```

Comment:
Why instead of expanding the patterns on `gnoPackagePathsFromPattern` and after that, call this function and create the `SubPkg`s, create the SubPkgs directly?

Reply:
I don't want to overload `SubPkgsFromPaths()` with many responsibilities.

Just to keep things modular. 
- `gnoPackagePathsFromPattern()`: resolves pattern and return paths
-  `SubPkgsFromPaths ()` takes path and return Subpkgs

I plan to use both functions independently in later PRs. Can we please keep it current way? It can save me from going back and forth in future PRs.